client/src/Pages/Game/index.tsx

```diff
@@ -85,11 +85,17 @@ const Game: React.FC = () => {
 
     setTimeout(() => {
       Konva.Image.fromURL(randomChar.image, function (image: any) {
+
+        // Scaling image based on original size. Currently uses a hacky way to do it but made the images more consistent in size
+        var imageHeight = image.attrs.image.height
+        var desiredHeight = 169
+        var scale = desiredHeight/imageHeight
```

Comment:
These can all be `const` as they are not being reassigned. For reassigned variables you can use `let`